src/Chapter2.hs

```diff
@@ -741,7 +755,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains ::  Int -> [[Int]] -> [[Int]]
+contains _ [] = []
```

Comment:
You don't need this extra special case, because `filter` returns empty list on empty list

Reply:
Noted!